wifiphisher/common/phishingpage.py

```diff
@@ -317,6 +317,33 @@ def get_templates(self):
 
         return self._templates
 
+    def search_template(self, name):
+        """
+        Check the directory for config and html
+        :param self: A TemplateManager object
+        :param name: A directory name
+        :type self: A TemplateManager object
+        :return: True or False
+        :rtype: None
+        """
+        config = False
+        html = False
+        # directory files
+        dir_files = os.listdir(os.path.join(self._template_directory, name))
+        # check all files in directory on...
+        for files in dir_files:
+                    # html
+            if files.find(".html", 0) != -1:
```

Comment:
can `find` recursively find all the files with `.html` ?

Reply:
Nope

Reply:
And also the `config.ini`. I'm asking this question because if in the future if we want to change our directory structure then this function still can be used transparently.